lib/wovnrb/html_replacers/link_replacer.rb

```diff
@@ -13,26 +13,53 @@ def initialize(store, pattern, headers)
       @headers = headers
     end
 
+
     def replace(dom, lang)
+      base_url = base_href(dom)
+
       dom.xpath('//*[match(.)]', MultiTagMatcher.new).each do |node|
         next if wovn_ignore?(node)
 
         href = node.get_attribute('href')
         next if href =~ /^\s*\{\{.+\}\}\s*$/
         next if href =~ /^\s*javascript:/i
         next if is_file?(href)
-        new_href = lang.add_lang_code(href, @pattern, @headers)
+
+        new_href = href
+        new_href = adjust_link_by_base(new_href, base_url) if base_url
+        new_href = lang.add_lang_code(new_href, @pattern, @headers)
+
         node.set_attribute('href', new_href)
       end
     end
 
+    private
+
+    def adjust_link_by_base(href, base_url)
+      return href if href =~ /^\//            # absolute path
+      return href if href =~ /^http(s?):\/\// # full url
+
+      File.join(base_url, href)
```

Comment:
Maybe we should use `Addressable::URI.join` here too?

Reply:
I tried. It does not work